explorer/interpreter/resolve_names.cpp

```diff
@@ -277,13 +293,20 @@ auto NameResolver::AddExposedNames(const Declaration& declaration,
 auto NameResolver::ResolveNames(Expression& expression,
                                 const StaticScope& enclosing_scope)
     -> ErrorOr<std::optional<ValueNodeView>> {
+  if (trace_stream_->is_enabled()) {
+  }
```

Comment:
Looks like this is left behind after some edits?

Reply:
Sorry about that, will remove in the next commit.